applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -28,65 +28,77 @@ config AUDIO_FRAME_DURATION_US
 	default 7500 if AUDIO_FRAME_DURATION_7_5_MS
 	default 10000 if AUDIO_FRAME_DURATION_10_MS
 	help
-		Audio frame duration in µs
+	  Audio frame duration in µs
+
+choice AUDIO_SAMPLE_RATE
+	prompt "Audio sample rate"
+	default AUDIO_SAMPLE_RATE_48000_HZ if BT_AUDIO_BROADCAST_RECOMMENDED || BT_AUDIO_UNICAST_RECOMMENDED
+	default AUDIO_SAMPLE_RATE_16000_HZ if BT_AUDIO_BROADCAST_16_2_1 || BT_AUDIO_BROADCAST_16_2_2 || BT_AUDIO_UNICAST_16_2_1
+	default AUDIO_SAMPLE_RATE_24000_HZ if BT_AUDIO_BROADCAST_24_2_1 || BT_AUDIO_BROADCAST_24_2_2 || BT_AUDIO_UNICAST_24_2_1
+	help
+	  This will be selected based on the BAP settings
+
+config AUDIO_SAMPLE_RATE_16000_HZ
+	bool "16 kHz"
+	depends on AUDIO_SOURCE_I2S
+
+config AUDIO_SAMPLE_RATE_24000_HZ
+	bool "24 kHz"
+	depends on AUDIO_SOURCE_I2S
+
+config AUDIO_SAMPLE_RATE_48000_HZ
+	bool "48 kHz"
+endchoice
 
 config AUDIO_SAMPLE_RATE_HZ
-	int "Sample rate Hz"
-	range 48000 48000
-	default 48000
+	int
+	default 16000 if AUDIO_SAMPLE_RATE_16000_HZ
+	default 24000 if AUDIO_SAMPLE_RATE_24000_HZ
+	default 48000 if AUDIO_SAMPLE_RATE_48000_HZ
 	help
-		For now 48 kHz is the only sample rate supported
+	  I2S supports 16, 24, and 48 kHz sample rates for both input and output.
+	  USB support only 48 kHz for input.
```

Comment:
supports

Reply:
Will add an 's' after 'USB support' to match